README.md

```diff
@@ -13,3 +15,26 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+## Solution
+
+**_List 3 reasons why asymptotic analysis may be misleading with respect to actual performance in practice._**
+
+1. Asymptotic analysis focuses on very large input sizes. This means input sizes which approach infinity, but in practice input sizes can be much smaller in alot of situations. An algorithm with better asymptotic complexity may have larger overhead that makes it perform worse for smaller input sizes. This is clear when comparing the graph of diferent time complexities. A polynomial or linear algorithm can perform better with a smaller input size then its logarithmic counterpart. Then as the input gets larger it will change and now the logarithmic complexity will be best.
```

Comment:
But we've analyzed algorithms that have their asymptotic behaviour for small input sizes.